lib/Echidna/Solidity.hs

```diff
@@ -120,12 +120,12 @@ contracts fp = let usual = ["--solc-disable-warnings", "--export-format", "solc"
                   (\sa -> if null sa then [] else ["--solc-args", sa])
         fps = toList fp
         compileOne :: (MonadIO m, MonadThrow m, MonadReader x m, Has SolConf x) => FilePath -> m [SolcContract]
-        compileOne x = maybe (throwM CompileFailure) (pure . toList . fst) =<< liftIO (do
+        compileOne x = maybe (throwM $ CompileFailure "" "could not read combined_solc.json") (pure . toList . fst) =<< liftIO (do
```

Comment:
I think this might be a bit misleading since this is not the actual stderr. Maybe a new data constructor to match on for `SolException`?